docs/development/release.md

```diff
@@ -2,8 +2,11 @@
 
 We use [electron-builder](https://github.com/electron-userland/electron-builder) to package and distribute the application. Which makes really easy doing that.
 
-1. Change the app version on [app/package.json](https://github.com/sqlectron/sqlectron-gui/blob/master/app/package.json#L3) based on [SemVer](http://semver.org/).
-1. Create a new **DRAFT** [release](https://github.com/sqlectron/sqlectron-gui/releases) with the same version of the previous step.
-1. On the next CI build it will upload the installers to this release. This allows testing the installers before release.
-1. Once the [release](https://github.com/sqlectron/sqlectron-gui/releases) is ready just publish it.
-1. Update [Homebrew Cask](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask).
+1. Merge the chagnes to the main branch.
+1. On Github open the Build action for the last build and test the app, it will have uploaded the artificats for each OS.
+1. Once you confirm the app is working as expected, then it is time to start the release process.
+1. Run `npm version <major|minor|patch>` based on the level of the changes being deployed. It will set a new version in the package.json and also create a new git tag for that.
+1. Run `git push --follow-tags` to push those changes to the remote git repository.
```

Comment:
while technically true on what's happening, given the other steps mention GitHub, might as well as just mention it here for clarity on where to look.
```suggestion
1. Run `git push --follow-tags` to push those changes to GitHub.
```